sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
```

Comment:
Please run `eng\scripts\Export-API.ps1 mobilenetwork` to generate the apiview file which is required for releasing a package.